### ISSUE TITLE
No image, video, or audio found 

### ISSUE THREAD

**Issue by nascentt** [Seklfreak/discord-image-downloader-go#184]

**Comment by drachedachte1** [comment]
Are you sure your acc ur using haves access to the channel?


Unofficial support server
https://discord.gg/8Gw3JQS

**Comment by nascentt** [comment]
That's correct. I have access, and the program was working perfectly with the same channels last week.
